### PR TITLE
fix: sets correct default value for composition strength.

### PR DESCRIPTION
**Pull request by jaydrennan**
also corrects positional argument error by requiring _imagine_cmd to take key word arguments.

### CODE REVIEW

File: imaginairy/api/generate_refiners.py [hunk 1]
```diff
@@ -229,7 +229,7 @@ def latent_logger(latents):
                 result_images["composition-upscaled"] = comp_image
                 composition_strength = prompt.composition_strength
                 # work around until bug is fixed in cli
-                composition_strength = 0.5
+                composition_strength = composition_strength
```
Comment (brycedrennan) [inline imaginairy/api/generate_refiners.py:232]: delete these lines

File: imaginairy/cli/edit.py [hunk 2]
```diff
@@ -102,45 +102,45 @@ def edit_cmd(
...
-        arg_schedules,
-        make_compilation_animation,
-        caption_text,
-        composition_strength,
+        ctx=ctx,
+        prompt_texts=prompt,
+        negative_prompt=negative_prompt,
+        prompt_strength=prompt_strength,
+        init_image=image_paths,
+        init_image_strength=image_strength,
```
Comment (jaydrennan) [inline imaginairy/cli/edit.py:110]: Are 109 and 110 mapped correctly?

File: imaginairy/api/generate_refiners.py (hunk 1 above)
Comment (jaydrennan) [reply imaginairy/api/generate_refiners.py]: done

File: imaginairy/cli/edit.py (hunk 2 above)
Comment (brycedrennan) [reply imaginairy/cli/edit.py:110]: yes

**Review by brycedrennan — APPROVED** [review]

**Pull request merged**